src/styles/components/Ayah.scss

```diff
@@ -94,7 +94,7 @@
   }
 }
 .word-font{
-  white-space: pre;
+  white-space: pre-line;
```

Comment:
Properties should be ordered color, font-size, line-height, overflow-wrap, white-space, width, word-break